.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: go
+go:
+    - 1.6.x
+    - 1.7.x
+    - 1.8.x
+    - tip
+
+sudo: required
+
+install:
+    - make install.tools
+
+before_script:
+    - export PATH=$HOME/gopath/bin:$PATH
```

Comment:
I'm not familiar with travis. I should have enabled travis for this project, we could try.